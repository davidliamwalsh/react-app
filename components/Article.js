import { Component } from 'react'
import Link from 'next/link'

class Article extends Component {
  render () {
    return <li className="c-articles__list">
      <img src="/static/article.png" className="c-articles__list--hero" />
      <div className="c-article__list__inner">
        <h4 className="c-articles__list__inner--title">Test Title</h4>
        <p className="c-articles__list__inner--body">Thus, when a URL is loaded straight (without click on the respective menu item), how do I catch this situation and highlight the respective menu item?</p>
        <p className="c-articles__list__inner--date">Posted on import date<br />
          <Link href="">
            <a>view more</a>
          </Link>
        </p>
      </div>
    </li>
  }
}

export default Article