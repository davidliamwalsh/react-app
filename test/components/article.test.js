import Article  from '../../components/Article'

describe('<Article />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <Article />
    )

    expect(component).toMatchSnapshot()
  })
}) 