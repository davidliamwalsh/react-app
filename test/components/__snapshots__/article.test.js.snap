// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Article /> it renders correctly 1`] = `ShallowWrapper {}`;